worlds/aquaria/docs/fr_Aquaria.md

```diff
@@ -0,0 +1,65 @@
+# Aquaria
+
+## Où se trouve la page des paramètres ?
+
+La [page des paramètres du joueur pour ce jeu](../player-settings) contiens tous
+les paramètres dont vous avez besoin pour configurer et exporter le fichier.
+
+## Quel est l'effet de la randomisation sur ce jeu ?
+
+Les localisations du "Ransomizer" sont:
+
+- tous les bulbes musicaux;
+- toutes les urnes de Mithalas;
+- toutes les caisses de la cité engloutie;
+- les localisations des trésors de collections (incluant les oeufs d'animaux de compagnie et les costumes);
+- Battre Simom dit;
+- La caverne de Li;
+- Les tortues de transportation (transturtle);
+- Localisation ou on obtient normalement les musiques,
+  * cristal de l'esprit Erulien,
+  * le mini-boss de la statue de l'énergie,
+  * battre le dieu de Mithalas,
+  * résoudre l'énigme de la caverne des poissons,
+  * battre le dieu Drunien,
+  * battre le dieu du soleil,
+  * détruire la cage de Li dans le corps,
+
+À noter que, contrairement au jeu original, lors de l'ouverture d'un bulbe musical, d'une urne de Mithalas ou 
+d'une caisse de la cité engloutie, aucun objet n'en sortira. La localisation représentée par l'objet ouvert est reçue
+dès l'ouverture.
+
+Les objets pouvant être obtenus sont:
+- les recettes (permettant d'apprendre les recettes*);
+- certains ingrédients;
+- le Wok (la troisième assiette permettant de cuisiner avec trois ingrédients n'importe où);
+- Tous les trésors de collection (incluant les oeufs d'animal de compagnie et les costumes);
+- Li et la musique de Li;
+- Toutes les musiques (autre que la musique de Li puisque cette dernière est apprise en obtenant Li);
+- Les localisations de transportation.
+
+Il y a également l'option pour mélanger les ingrédients obtenus en éliminant des montres, des poissons ou des plantes. 
+
+*À noter que, contrairement au jeu original, il est impossible de cuisiner une recette qui n'a pas préalablement
+été apprise en obtenant un repas en tant qu'objet. À noter également que les ennemies et plantes qui
+donnent un repas dont la recette n'a pas préalablement été apprise vont donner les ingrédients de cette
+recette.
+
+## Quel est le but de DLC Quest ?
```

Comment:
It looks like this refers to DLC Quest, which is the wrong game

Reply:
Oups. I use DLC Quest as template for the french version of the documentation. I forgot to rename this line. Just replace the DLC Quest by Aquaria. I can do it, but I am not sure if I actually can considering that you start the review process.

Reply:
You can change it during the review process, that's fine. Just push the commits to the same branch this PR is from

Reply:
Ok. I have push a new commit to the branch.